tests/experiment/tools/test_rest.py

```diff
@@ -727,7 +727,7 @@ def test_project_fields_instruments(brain_vol: BrainVol) -> None:
     assert len(project) > 0
     # fv = project['field_values']
     fv = project
-    assert type(fv) is list
+    assert isinstance(fv, list)
```

Comment:
well, tests could have remained to be more restrictive via ` is ` but I think it is ok also as this